app/kuma-cp/cmd/root.go

```diff
@@ -41,6 +50,10 @@ func newRootCmd() *cobra.Command {
 	}
 	// root flags
 	cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", kuma_log.InfoLevel.String(), kuma_cmd.UsageOptions("log level", kuma_log.OffLevel, kuma_log.InfoLevel, kuma_log.DebugLevel))
+	cmd.PersistentFlags().StringVar(&args.outputPath, "output-path", args.outputPath, "path of output rotating file")
+	cmd.PersistentFlags().IntVar(&args.maxBackups, "max-backups", 1000, "maximum number of old log files to retain")
+	cmd.PersistentFlags().IntVar(&args.maxSize, "max-size", 100, "maximum size in megabytes of a log file before it gets rotated")
+	cmd.PersistentFlags().IntVar(&args.maxAge, "max-age", 30, "maximum number of days to retain old log files based on the timestamp encoded in their filename")
```

Comment:
I think we should prefix those options with `log-` so it's
```
cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", kuma_log.InfoLevel.String(), kuma_cmd.UsageOptions("log level", kuma_log.OffLevel, kuma_log.InfoLevel, kuma_log.DebugLevel))
cmd.PersistentFlags().StringVar(&args.outputPath, "log-output-path", args.logOutputPath, "path to the file that will be filled with logs")
cmd.PersistentFlags().IntVar(&args.maxBackups, "log-max-retained-files", 1000, "maximum number of the old log files to retain")
cmd.PersistentFlags().IntVar(&args.maxSize, "log-max-size", 100, "maximum size in megabytes of a log file before it gets rotated")
cmd.PersistentFlags().IntVar(&args.maxAge, "log-max-age", 30, "maximum number of days to retain old log files based on the timestamp encoded in their filename")
```

Reply:
done